client/hostvolumemanager/host_volumes.go

```diff
@@ -43,97 +45,37 @@ type Config struct {
 	UpdateNodeVols HostVolumeNodeUpdater
 }
 
+// HostVolumeManager executes plugins, manages volume metadata in client state,
+// and registers volumes with the client node.
 type HostVolumeManager struct {
 	pluginDir      string
 	sharedMountDir string
 	stateMgr       HostVolumeStateManager
 	updateNodeVols HostVolumeNodeUpdater
+	builtIns       map[string]HostVolumePlugin
 	log            hclog.Logger
 }
 
+// NewHostVolumeManager includes default builtin plugins.
 func NewHostVolumeManager(logger hclog.Logger, config Config) *HostVolumeManager {
-	// db TODO(1.10.0): document plugin config options
 	return &HostVolumeManager{
 		pluginDir:      config.PluginDir,
 		sharedMountDir: config.SharedMountDir,
 		stateMgr:       config.StateMgr,
 		updateNodeVols: config.UpdateNodeVols,
-		log:            logger.Named("host_volume_manager"),
+		builtIns: map[string]HostVolumePlugin{
+			HostVolumePluginMkdirID: &HostVolumePluginMkdir{
+				ID:         HostVolumePluginMkdirID,
+				TargetPath: config.SharedMountDir,
+				log:        logger.With("plugin_id", HostVolumePluginMkdirID),
```

Comment:
Nitpick: this logger won't be named `host_volume_manager` but the loggers we instantiate for the external plugins will, because they get cloned from `hvm.log.With`. Maybe we should hoist the `logger.Named` to the top of this method and then have the mkdir plugin call `.With` on that?